e2e/pages/filing-app/unavailableApis.spec.ts

```diff
@@ -0,0 +1,208 @@
+      await expect(page.locator('h2')).toContainText('Select a file to upload');
+      const fileChooserPromise = page.waitForEvent('filechooser');
+      await page.getByLabel('Select a .csv file to upload').click();
+      const fileChooser = await fileChooserPromise;
+      await fileChooser.setFiles(
+        path.join(
+          __dirname,
+          '../../test-data/sample-sblar-files/sbl-validations-all-pass-small.csv',
+        ),
+      );
+      await expect(page.getByText('File upload in progress')).toBeVisible();
+      await expect(page.getByText('File upload successful')).toBeVisible({
+        timeout: 10_000,
+      });
+      await expect(page.getByText('Validation checks in progress')).toBeVisible(
+        {
+          timeout: 10_000,
+        },
+      );
+      await expect(
+        page.getByText('Warnings were found in your file'),
+      ).toBeVisible({ timeout: 60_000 });
+    });
+
+    // Continue to next page
+    await test.step('Navigate: Resolve errors (syntax)', async () => {
+      await page.waitForSelector('#nav-next');
+      await page.waitForTimeout(500);
+      await page.getByRole('button', { name: 'Continue to next step' }).click();
+    });
+  });
+
+  // Resolve errors page
+  await test.step('Resolve errors page', async () => {
+    await expect(page.locator('h1')).toContainText('Resolve errors (syntax)');
+
+    // Block API Call: **/submisions/latest
+    await test.step('Block API: /submissions/latest', async () => {
+      await blockApi(page, '**/submissions/latest', true);
+    });
+
+    // Confirm Error Boundary
+    await test.step('Error Boundary', async () => {
+      await page.reload();
+      await page.waitForSelector('h1', { state: 'visible' });
+      await expect(page.locator('h1')).toContainText(
+        'An unknown error occurred',
+      );
+    });
+
+    // Unblock API Call
+    await test.step('Unblock API', async () => {
+      await blockApi(page, '**/submissions/latest', false);
+      await expect(page.locator('h1')).toContainText('Resolve errors (syntax)');
+    });
+
+    // Navigate: Resolve errors (logic)
+    await test.step('Navigate: Resolve errors (logic)', async () => {
+      await page.getByRole('button', { name: 'Continue' }).click();
+      await expect(page.locator('h1')).toContainText('Resolve errors (logic)');
+    });
+
+    await test.step('Navigate: Review Warnings', async () => {
+      await page.getByRole('button', { name: 'Continue to next step' }).click();
+    });
+  });
+
+  // Review warnings page
+  await test.step('Review warnings page', async () => {
+    await expect(page.locator('h1')).toContainText('Review warnings');
+
+    // Block API Call: **/v1/institutions/
+    await test.step('Block API: /v1/institutions', async () => {
+      await blockApi(page, new RegExp(`.*?/v1/institutions/${testLei}`), true);
+    });
+
+    // Confirm Error Alert
+    await test.step('Error Alert', async () => {
+      test.setTimeout(120_000);
```

Comment:
Yeah, the curse/blessing of having a bunch of retries. These tests are really great, but it will be a bummer having long waits if we end up running these tests often in CI. Can we make that ticket to make retries a variable affected by the environment, so when we're testing we prevent or limit retries? 

Then maybe let's add a comment above these long retries with a link to the ticket ([like this one](https://github.com/cfpb/sbl-frontend/blob/main/src/components/BetaAndLegalNotice.tsx#L27-L28)) to remind ourselves to adjust these timeouts when we're done?

Reply:
Created issue #916 and added ToDo comments.